src/app/api/quality_matrix/quality_matrix.py

```diff
@@ -49,8 +49,175 @@ def all_sources() -> dict[str, int]:
     )
 
 
+def default_properties() -> list[str]:
+    return [
+        "virtual:collection_id_primary",
+        "virtual:collection_id",
+        "ccm:collection_proposal_status",
+        "virtual:portal_type",
+        "ccm:oeh_quality_correctness",
+        "ccm:published_handle_id",
+        "sys:node-uuid",
+        "sys:node-uuid",
+        "cm:versionLabel",
+        "virtual:search_tasks",
+        "virtual:author",
+        "virtual:time_after_modified",
+        "virtual:time_after_created",
+        "virtual:editorial_exclusion",
+        "virtual:editorial_taxonid",
+        "virtual:editorial_file_type",
+        "virtual:editorial_edit_type",
+        "virtual:editorial_filter",
+        "virtual:editorial_status",
+        "virtual:publish_location",
+        "virtual:data_privacy",
+        "virtual:newsletter",
+        "virtual:email",
+        "ccm:lifecyclecontributer_publisher",
+        "ccm:replicationsource",
```

Comment:
why are some of the entries here duplicated? Is that intentional?

Reply:
Oh, good spotting. Not intended - bizarre. Fixed it